AdyenComponents/UPI/UPIComponentDetails.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+
+/// Contains the details supplied by the UPI based component.
```

Comment:
"UPI based" ? 🤔

Reply:
Fixed in this commit: AdyenComponents/UPI/UPIComponentDetails.swift